Description: by default use octave-cli rather than octave
 'octave' can randomly decide to load its GUI rather than the CLI depending
 on environmental factors. We cannot stdin control the GUI, so there is no
 point in introducing this instability but instead call octave-cli directly
 since it is the only possibly operation mode we support.

 (NB there also is a --no-gui switch which for me doesn't appear to work for
  some reason, so octave-cli directly seems the most reliable approach to
  this)
Author: Harald Sitter <sitter@kde.org>
Origin: vendor
Forwarded: https://git.reviewboard.kde.org/r/125646/
Reviewed-by: Rohan Garg <rohan@kde.org>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/src/backends/octave/octavebackend.kcfg b/src/backends/octave/octavebackend.kcfg
index be3f4f4..286910b 100644
--- a/src/backends/octave/octavebackend.kcfg
+++ b/src/backends/octave/octavebackend.kcfg
@@ -8,7 +8,7 @@
   <group name="OctaveBackend">
     <entry name="Path" type="Url">
       <label>Path to the Octave executable</label>
-      <default code="true">QUrl::fromLocalFile(QStandardPaths::findExecutable( QLatin1String("octave") ))</default>
+      <default code="true">QUrl::fromLocalFile(QStandardPaths::findExecutable( QLatin1String("octave-cli") ))</default>
     </entry>
     <entry name="integratePlots" type="Bool">
       <label>Integrate Plots into the Worksheet</label>
-- 
2.1.4

